File: Level1/문자열내p와y의개수/solution.py
def solution(s):
    answer = True

    p_num = 0
    y_num = 0

    for x in range(len(s)):
        if s[x] == 'p' or s[x] == 'P':
            p_num += 1
        elif s[x] == 'y' or s[x] == 'Y':
            y_num += 1

    if p_num != y_num:
        return False
    else:
        return True

    return True

File: Level1/문자열내p와y의개수/solution.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    bool answer = true;
    int p_num = 0;
    int y_num = 0;

    for (int i = 0; i < s.size(); i++){
        if (s[i] == 'p' or s[i] == 'P')
            p_num += 1;
        else if (s[i] =='y' or s[i] == 'Y')
            y_num += 1;
    }


    if (p_num == y_num)
        answer = true;
    else
        answer = false;

    return answer;
}